storagenode/peer.go

```diff
@@ -99,9 +99,10 @@ type Peer struct {
 	}
 
 	Storage2 struct {
-		Trust    *trust.Pool
-		Store    *pieces.Store
-		Endpoint *piecestore.Endpoint
+		Trust     *trust.Pool
+		Store     *pieces.Store
+		Endpoint  *piecestore.Endpoint
+		Inspector *piecestore.Inspector
```

Comment:
I guess we can remove the other reference.

Reply:
true